internal/server/tls.go

```diff
@@ -26,21 +23,47 @@ func tlsConfigFromOptions(
 	tlsCacheDir string,
 	opts TLSOptions,
 ) (*tls.Config, error) {
-	// TODO: print CA fingerprint when the client can trust that fingerprint
+	// TODO: how can we test this?
```

Comment:
Not really sure how to test the ACME integration, but it wasn't tested before and it's still not documented as a supported feature, so probably not a requirement for this PR.